packs/chatops/actions/templates/default.j2

```diff
@@ -0,0 +1,44 @@
+{%- macro serialize(iter, prefix='') -%}
```

Comment:
Jinja template? Because fitting something like that into the python code and then struggling with formatting is not fun =)